examples/packages/pandas_example.py

```diff
@@ -0,0 +1,59 @@
+"""
+You can store Pandas DataFrame objects in Hazelcast cluster and retrieve with low-latency.
+There are multiple serialization methods for DataFrame objects.
+Hazelcast uses pickle serialization as default for various Python objects including DataFrame.
+So, you can simply put your data directly using default pickle serialization without any conversions.
+
+patients.put(1, df)
+
+Alternatively, you can convert your DataFrame to JSON, CSV or a dict using to_json(), to_csv() and to_dict() methods.
+Note that JSON or CSV serializations returns string representation of DataFrame in requested format.
+
+Convert to CSV: df = df.to_csv()
+Convert to JSON: df = df.to_json()
+Convert to dict: df = df.to_dict()
+
+If you prefer to use these converted representations, you need to re-create DataFrame object since they are not stored
+as DataFrame object in Hazelcast cluster. For CSV, JSON and dict conversions, use following methods while retrieving:
+
+Create from CSV: df = pd.from_csv(StringIO(patients.get(1)))
+Create from JSON: df = pd.read_json(patients.get(1))
+Create from dict object: df = pd.DataFrame(patients.get(1))
+
+In addition to methods above, you can write your own custom serializer for DatFrame objects. For more information about
+Pyton client serialization methods, see https://hazelcast.readthedocs.io/en/stable/serialization.html#
+"""
+
+import hazelcast
+from hazelcast.core import HazelcastJsonValue
+from matplotlib import pyplot as plt
+import np as np
```

Comment:
This should be `import numpy as np`.

Reply:
Thanks, fixed now https://github.com/hazelcast/hazelcast-python-client/pull/604/commits/e1d903640dc9d5c3146b084577695a1b0d5bcc1f.